device-registry/src/main/scala/org/genivi/sota/device_registry/db/DeviceRepository.scala

```diff
@@ -42,7 +42,11 @@ object DeviceRepository {
   // scalastyle:on
   val devices = TableQuery[DeviceTable]
 
-  def list(ns: Namespace): DBIO[Seq[Device]] = devices.filter(_.namespace === ns).result
+  def list(ns: Namespace, offset: Long, limit: Long): DBIO[Seq[Device]] =
```

Comment:
would be good to have this return something different, like `DeviceSearchResult` or something, that would include the offset, total items and the devices for the current offset.